src/dbnode/persist/fs/retriever.go

```diff
@@ -534,67 +567,62 @@ func (r *blockRetriever) Stream(
 	// the data. This means that even though we're returning nil for error
 	// here, the caller may still encounter an error when they attempt to
 	// read the data.
-	return req.toBlock(), nil
+	return nil
 }
 
-func (r *blockRetriever) StreamIndexChecksum(
+func (req *retrieveRequest) toBlock() xio.BlockReader {
```

Comment:
Did this need to be moved here? Should it not be close to the other `retrieveRequest` declared methods?

Reply:
Ah I see it was not grouped with the other declared methods before, can we move it next to other `retrieveRequest` declared methods?

Reply:
Good call